quantum/process_keycode/process_auto_shift.c

```diff
@@ -178,7 +178,11 @@ static bool autoshift_press(uint16_t keycode, uint16_t now, keyrecord_t *record)
 #        ifdef TAPPING_TERM_PER_KEY
         get_tapping_term(autoshift_lastkey, record)
 #        else
+#            ifdef DYNAMIC_TAPPING_TERM_ENABLE
+        g_tapping_term
+#            else
         TAPPING_TERM
+#            endif
```

Comment:
```suggestion
        g_tapping_term
```
The variable `g_tapping_term` is not behind any `#ifdef` (other than `#ifndef NO_ACTION_TAPPING`) so even if `DYNAMIC_TAPPING_TERM_ENABLE` is off, `g_tapping_term` is still a reference to the `TAPPING_TERM` defined by the user (or 200 by default);